File: class_objects.py

class Food:
    # private
    __cost= None
    __type= None
    def __init__(self, cost, type) -> None:
        self.__cost= cost
        self.__type= type 
        
    def __str__(self) -> str:
        print("hello")
        return f"{self.__type} of food costs {self.__cost} BDT"
    

cake= Food(20, "dessert" )
print(cake)

File: class_objects.cpp
#include<iostream>
#include<bits/stdc++.h>
using namespace std;


class Food{
    private:
        int cost;
        string type;
    public:
        Food(int cost, string type){
            this->cost= cost;
            this->type=type;
        }
        void display(){
            cout<<type<< " type of food costs "<<cost<<" BDT "<<endl;
        }

};

int main(){

    Food cake(20, "dessert");
    cake.display();

   
return 0;
}